### DISCUSSION TITLE
About folders deletion

### DISCUSSION THREAD

**Discussion by baannng** [FreshRSS/FreshRSS#3509] [General]
Hi

Which folders (and files) can I delete from my server after installation?
.github
app
cli
data
Docker
docs
lib
tests

**Comment by Alkarex** [comment]
I advise not to delete files manually. It is really not much space, they will come back when you update anyway, and the list might change.
Anyway, this was already answered: https://github.com/FreshRSS/FreshRSS/issues/3182#issuecomment-695201067
